pennylane/wires.py

```diff
@@ -495,15 +495,15 @@ def unique_wires(list_of_wires):
         return Wires(tuple(unique), _override=True)
 
     def union(self, other):
-        """Return the union of the current Wires object and either another Wires object or an
-        iterable that can be interpreted like a Wires object e.g., List.
+        """Return the union of the current :class:`~.Wires` object and either another :class:`~.Wires` object or an
+        iterable that can be interpreted like a :class:`~.Wires` object, e.g., a ``list``.
```

Comment:
This is not blocking, but you might not want to use "iterable" in this case since it's not completely accurate. It technically takes in `Union[Iterable[Hashable], Hashable]`.

Not a huge fan of using `WiresLike` either since you'd have to go and look at the codebase itself to find information about it. Maybe just write  "... or an [Iterable[Hashable, Hashable] that can be..."
```suggestion
        `WiresLike` that can be interpreted like a :class:`~.Wires` object, e.g., a ``list``.
```

Reply:
And all other instances where it says "...iterable that can be interpreted..."